CMakeLists.txt

```
@@ -29,7 +29,7 @@ endif()
 
 project(microxrcedds VERSION "2.0.0")
 
-set(_client_tag develop)
+set(_client_tag feature/client_api_update)
```

```suggestion
set(_client_tag develop)
```

Revert when https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/205 merged